modules/bitgo/src/v2/coins/near.ts

```diff
@@ -155,10 +158,26 @@ export class Near extends BaseCoin {
    * Explain/parse transaction
    * @param params
    */
-  explainTransaction(
+  async explainTransaction(
     params: ExplainTransactionOptions,
-  ): Promise<any> {
-    throw new MethodNotImplementedError('Near recovery not implemented');
+  ): Promise<NearTransactionExplanation> {
+    const factory = accountLib.register(this.getChain(), accountLib.Near.TransactionBuilderFactory);
+    let rebuiltTransaction;
```

Comment:
reorder code here

Reply:
If I declare `rebuiltTransaction` as a constant inside the try block, I cannot use it later in line 175 because it is outside the scope of the try, that is why it was defined outside.